### PR TITLE
draft: Fix/mip new name: mip-dna 

### PR DESCRIPTION
**Pull request by hassanfa**
This PR adds mip-dna to list of acceptable workflows. and removes mip.

**Review:**
- [x] code approved by Chiara
- [x] tests executed by @hassanfa 
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: vogue/commands/load/bioinfo/bioinfo_raw.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
         It can be specified multiple times.''')
 @click.option('-w',
               '--analysis-workflow',
-              type=click.Choice(['mip', 'balsamic', 'microsalt']),
+              type=click.Choice(['mip-dna', 'mip-rna', 'balsamic', 'microsalt']),
```
Comment (northwestwitch) [inline vogue/commands/load/bioinfo/bioinfo_raw.py:53]: Looks good

**Review by northwestwitch — APPROVED** [review]

**Comment by hassanfa** [comment]
Alright, ready to review!

File: vogue/commands/load/bioinfo/bioinfo_raw.py (hunk 1 above)
Comment (hassanfa) [reply vogue/commands/load/bioinfo/bioinfo_raw.py]: Not so fast! I have more files to add! adding tests as well :-)

**Comment by hassanfa** [comment]
MIP page loads successfully:
image

File: vogue/commands/load/bioinfo/bioinfo_raw.py (hunk 1 above)
Comment (northwestwitch) [reply vogue/commands/load/bioinfo/bioinfo_raw.py]: Ah right, when you asked for my review all the other files were not there! :)

**Review by mayabrandi — COMMENTED** [review]
Looks all good to me, but would be nice to double check tat the MIP_DNA_PICARD plots work after the change. Could you upload just a few samples and check that they show up in the MIP picard QC ?

**Comment by hassanfa** [comment]
@mayabrandi this PR is ready to go. See below test:

image

I load 8 cases from Stage, and they are the only ones in Stage-db (some of them below:
image

**Comment by mayabrandi** [comment]
yey! Sweet @hassanfa

**Review by mayabrandi — APPROVED** [review]
👍

**Pull request merged**